layerfetcher/layers.go

```diff
@@ -241,9 +241,14 @@ func (l *Logic) tortoiseBeaconReqReceiver(ctx context.Context, msg []byte) []byt
 	epoch := types.EpochID(util.BytesToUint32(msg))
 	l.log.WithContext(ctx).Debug("got tortoise beacon request for epoch %v", epoch)
 
-	beacon, ok := l.tbDB.GetTortoiseBeacon(epoch)
-	if !ok {
-		l.log.WithContext(ctx).Warning("cannot find tortoise beacon for epoch %v", epoch)
+	beacon, err := l.tbDB.GetTortoiseBeacon(epoch)
+	if errors.Is(err, database.ErrNotFound) {
+		l.log.WithContext(ctx).Debug("still no tortoise beacon for epoch %v", epoch)
```

Comment:
Is there a case where a request to a beacon that is not yet calculated can be made?
`ErrNotFound` doesn't necessarily mean that we didn't calculate the beacon yet- it could mean that something is wrong